Header                   header
float64[16]              radio
float64[10]              motor_torque
float64[14]              encoder_position
float64[14]              encoder_velocity
geometry_msgs/Quaternion orientation
geometry_msgs/Vector3    angular_velocity
geometry_msgs/Vector3    linear_velocity
geometry_msgs/Vector3    linear_acceleration
float64[2]               q_achilles
float64[2]               dq_achilles
float64[2]               contact
